src/assets/styles/globals.css

```diff
@@ -11,62 +11,11 @@ body {
   box-sizing: border-box;
 }
 
-.container {
```

Comment:
old css styles that aren't used.

the ones that were added are for VACMS-15511